_includes/releases/cloud/2023-05-05.md

```diff
@@ -0,0 +1,3 @@
+## May 5, 2023
```

Comment:
You need to add a row in `_data/cloud_releases.yml`. If you happen to have a SHA, include it. Otherwise, put `null`.